service_management/azure/test/integration/storage_management/storage_management_test.rb

```diff
@@ -16,170 +16,170 @@
 
 describe Azure::StorageManagementService do
 
-  util = Class.new.extend(Azure::Core::Utility)
   subject { Azure::StorageManagementService.new }
-  affinity_name = util.random_string('affinity-group-', 10)
-  Azure::BaseManagementService.new.create_affinity_group(
-    affinity_name,
-    'West US',
-    'Label Name'
-  )
-  StorageName =  util.random_string('storagetest', 10)
+  affinity_name = 'testaffinitygroup44938'
```

Comment:
Should we use pattern let(:affinity_name) { 'testaffinitygroup44938' } to lazy load resources?